README.md

```diff
@@ -39,12 +39,16 @@ It is likely that you will have your own Kubernetes cluster (e.g. local, GKE, EK
 
 #### Option 1: Minikube
 
-[Install minikube](https://k8s-docs.netlify.app/en/docs/tasks/tools/install-minikube/) (if needed), then start a single node cluster:
+[Install minikube](https://k8s-docs.netlify.app/en/docs/tasks/tools/install-minikube/) (if needed), then start a 4-node cluster:
 
 ```sh
-minikube start
+minikube start --nodes 4 --extra-config=apiserver.service-node-port-range=8081-65535
 ```
 
+This command starts minikube with 4 nodes (1 control plane, 3 worker nodes) and will extend the NodePort range to include default ports for Redpanda services.
```

Comment:
I changed this to install a 4-node cluster rather than a single node, since the default external access method is via NodePort. So each node where Redpanda runs will open the same node, allowing access directly to each broker for external clients. But this only works if we have at least the same number of nodes as replicas in the statefulset, which is 3 by default.